### PR TITLE
Fix for querystring parsing

### PR DESCRIPTION
**Pull request by ece-mohammad**
- Fix for querystring parsing [issue](https://github.com/mattrobenolt/jinja2-cli/issues/116#issue-1932394171)
- a test for querystring parsing

### CODE REVIEW

**Comment by mattrobenolt** [comment]
Can you just remove all of the unrelated changes and the comment for why the change was made? That's quite odd to leave a comment like that.

File: jinja2cli/cli.py [hunk 1]
```diff
@@ -165,7 +165,11 @@ def _parse_qs(data):
         """
         dict_ = {}
         for k, v in urlparse.parse_qs(data).items():
-            v = map(lambda x: x.strip(), v)
+            # Changed:
+            # v = map(lambda x: x.strip(), v)
+            # Reason:
+            # >>> TypeError: object of type map has no len()
```
Comment (mattrobenolt) [inline jinja2cli/cli.py:171]:
```suggestion
```

File: jinja2cli/cli.py [hunk 2]
```diff
@@ -138,7 +138,7 @@ def as_dict(self):
 
 
 def _load_yaml():
-    from yaml import load, YAMLError
+    from yaml import YAMLError, load
```
Comment (mattrobenolt) [inline jinja2cli/cli.py:141]: Why was this changed?

File: jinja2cli/cli.py [hunk 3]
```diff
@@ -243,12 +248,8 @@ def _load_json5():
 
 
 def render(template_path, data, extensions, strict=False):
-    from jinja2 import (
-        __version__ as jinja_version,
-        Environment,
-        FileSystemLoader,
-        StrictUndefined,
-    )
+    from jinja2 import Environment, FileSystemLoader, StrictUndefined
```
Comment (mattrobenolt) [inline jinja2cli/cli.py:251]: Why was this changed?

**Review by mattrobenolt — CHANGES_REQUESTED** [review]

**Comment by ece-mohammad** [comment]
Sorry for the unrelated changes. I had a pre-commit hook with `black` and `isort` and it made those unrelated changes. 
As for the comment, I'll remove it, and make another PR.
